internal/namespaces/instance/v1/updated_commands.go

```diff
@@ -46,3 +48,81 @@ func updateInstancePlacementGroupGet(c *core.Command) {
 		},
 	}
 }
+
+func updateInstanceServerUpdate(c *core.Command) {
+
+	type instanceUpdateServerRequestCustom struct {
+		*instance.UpdateServerRequest
```

Comment:
I'm not sure we handle anonymous filed in args unmarshalling (maybe we should) but I don't think this will work as is.

Reply:
Indeed, it doesn't work. 

Do we want to add support for anonymous fields, or should we copy manually all the fields from `instance.UpdateServerRequest` to `instanceUpdateServerRequestCustom` ?

Reply:
Or should we use reflection to copy theses fields automatically ?

Reply:
This is handled now